src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { ResultsList, SearchBox } from './components';
+import './App.css';
+
+class App extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
```

Comment:
think this app needs a bit of restructuring. Will talk to you in person about it.